### ISSUE TITLE
Recommend dj-database-url

### ISSUE THREAD

**Issue by smn** [praekeltfoundation/ways-of-working#16]
Allows for easier ways of specifying dbs in travis, qa, production: https://github.com/kennethreitz/dj-database-url

**Comment by Rizziepit** [comment]
@smn there are other settings (e.g. email host and port) which could benefit from something similar. Would you recommend using `os.environ.get('SETTING_NAME', '')`?

**Comment by smn** [comment]
Yeah, let's use this ticket to discuss other variables we'd possibly want set & some possible sane ways of going about this.
